airflow/providers/facebook/ads/operators/ads.py

```diff
@@ -0,0 +1,119 @@
+#
```

Comment:
The file name should be called facebook_ads_to_gcs and should be in the `airflow/providers/google/cloud/operators` package.
More information: https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-21%3A+Changes+in+import+paths

Reply:
Will fix it according to the AIP. But what if tomorrow one would want integration with AWS or other cloud plf?

Reply:
Then he will create a new operator, which will not be called FacebookAdsTo**GCS**.

Reply:
Cool, but the hook is consumed from google/FacebookAdsToGCS/hooks ??

Reply:
Hook should be in airflow/providers/facebook/ads/hooks.